.github/workflows/sonar.yml

```diff
@@ -52,8 +50,16 @@ jobs:
 
       - name: Generate Unit test coverage for LLD & LLM
         run: |
-          pnpm desktop test:jest:coverage --coverageDirectory=desktop-coverage
-          pnpm mobile test:jest:coverage --coverageDirectory=mobile-coverage
+          pnpm desktop test:jest:coverage --collectCoverage=true
+          pnpm mobile test:jest:coverage --collectCoverage=true
```

Comment:
Do we need to use `--collectCoverage=true` as for both commands we use `--coverage` which is an alias to `collectCoverage=true`?

Reply:
you right, just added it to make sure unit test field in sonar take coverage per file .. just removed thanks